openehr-rm/src/test/java/com/nedap/archie/rm/composition/CompositionTest.java

```diff
@@ -35,7 +35,7 @@ public void testJsonStandardConfig() throws IOException {
     @Test
     public void testJsonLegacyConfig() throws IOException {
         Composition expected = parseJson("validation_composition_test.v0.json");
-        ArchieJacksonConfiguration config = ArchieJacksonConfiguration.createStandardsCompliant();
+        ArchieJacksonConfiguration config = ArchieJacksonConfiguration.createConfigForJavascriptUsage();
```

Comment:
Yes. I just found out that Intellij has interesting auto-complete features when the currently selected on is deprecated - it will just pick the next one. Not nice at all. Will fix :)